### PR TITLE
Bz1935027 must gather

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1935027

Preview: https://deploy-preview-109--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#using-must-gather_forklift

### CODE REVIEW

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ If a migration does not complete, you can create a new `Migration` CR, without c
 
 .Procedure
 
-. Obtain the SSL thumbprint of the vCenter host:
+. Obtain the SHA-1 fingerprint of the ESXi host:
```
Comment (fdupont-redhat) [inline documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc:27]: This is the fingerprint of the vCenter that we want, not ESXi !

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/using-must-gather.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* You must have the link:https://docs.openshift.com/container-platform/{ocp-version}/cli_reference/openshift_cli/getting-started-cli.html[{ocp} CLI (`oc`)] installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+. Run the `oc adm must-gather` command with the `--image` argument:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=quay.io/konveyor/forklift-must-gather:latest
```
Comment (fdupont-redhat) [inline documentation/doc-Migration_Toolkit_for_Virtualization/modules/using-must-gather.adoc:22]: The image name will be different downstream: `registry.redhat.io/rhmtv/rhmtv-must-gather:v2.0.0`
So, we probably need a variable here.

**Review by fdupont-redhat — CHANGES_REQUESTED** [review]

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc:27]: Fixed.

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/using-must-gather.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/doc-Migration_Toolkit_for_Virtualization/modules/using-must-gather.adoc:22]: Good idea

**Review by fdupont-redhat — APPROVED** [review]
LGTM

**Pull request merged**